modules/eks/alb-controller-ingress-group/remote-state.tf

```
@@ -0,0 +1,39 @@
+module "dns_delegated" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "0.22.3"
```

```suggestion
  version = "0.22.4"
```
plus all other occurrences